cockatrice/src/remoteclient.cpp

```diff
@@ -299,6 +343,30 @@ void RemoteClient::doRegisterToServer(const QString &hostname, unsigned int port
     setStatus(StatusRegistering);
 }
 
+void RemoteClient::doForgotPassword(const QString &hostname, unsigned int port, const QString &_userName, const QString &_email)
+{
+	doDisconnectFromServer();
+
+	userName = _userName;
+	email = _email;
```

Comment:
Well I disagree with not having the email verified. If a user has to type an email address it, it better be compared to the database email on file... otherwise I want to see this field removed from the asking question phase.